src/HelperPluginManager.php

```diff
@@ -219,7 +220,7 @@ class HelperPluginManager extends AbstractPluginManager
      * helper works fine as an invokable. The factory for doctype simply checks for the
      * config value from the merged config.
      *
-     * @var array
+     * @var array<array-key, callable|string>
```

Comment:
If this is inherited, I would suggest dropping the type declaration, for more precise type definitions to be inherited later on

Reply:
Got it!